woocommerce/wc-admin
# Try: Consume remote payment methods on frontend

This PR is a preliminary look at how that remote payment methods would be hooked up to the frontend.  This will help us better shape the schema required for various methods.

This PR does not (follow-ups):

* Include a data store for payment methods.
* Remove existing method files and clean up unused payment items.
* Handle enabled payment methods in the task list.
* Handle connection flows.

### Screenshots
<img width="694" alt="Screen Shot 2021-04-22 at 6 15 51 PM" src="https://user-images.githubusercontent.com/10561050/115793991-3175e380-a39b-11eb-8ccf-52ee8cfdf7be.png">
<img width="695" alt="Screen Shot 2021-04-22 at 6 15 45 PM" src="https://user-images.githubusercontent.com/10561050/115793993-320e7a00-a39b-11eb-9825-ea1f64a083ec.png">

### Detailed test instructions:

1. Install and activate https://github.com/joshuatf/remote-payments-tester
2. Note that `PayFast` is fetched and included under the payments task when the store country is set to South Africa.
3. Click `Set up` and note the `PayFast` fields are also included.
4. Optionally rebase with https://github.com/woocommerce/woocommerce-admin/pull/6866 for option saving to work as expected.
